### ISSUE TITLE
AZ-104 learn link not correct

### ISSUE THREAD

**Issue by tradexy** [MicrosoftDocs/feedback#2999]
AZ-104 learn link not correct
AZ-104: Deploy and manage Azure compute resources - on continue button

It needs to be changed to 

**To Reproduce**
Page https://docs.microsoft.com/en-gb/learn/paths/az-104-manage-compute-resources/
Link on continue button is https://docs.microsoft.com/en-gb/learn/modules/intro-to-azure-kubernetes-service/2-what-is-azure-kubernetes-service/

**Expected behavior**
It needs to go to this page 
https://docs.microsoft.com/en-gb/learn/modules/create-linux-virtual-machine-in-azure/

**Desktop (please complete the following information):**
 - Browser chrome, did not test other browsers

**Additional context**
Straight forward update required

**Comment by JasonWHowell** [comment]
@cryophobia  There's a broken link on your Kubernetes learn module. Please take a look. Thanks!

**Comment by cryophobia** [comment]
Hi @JasonWHowell - which link is broken on the Kubernetes module?

The reported behavior described by @tradexy is what is expected when a user completes modules in a learning path. The **Continue** button will pick up the next unit not yet completed, from the list of started but not yet completed modules, in the learning path based on the signed-in user's profile. For example, the first module in my profile that matches the criteria is Build Azure Resource Manager templates. This is the 7th module in this learning path.

However, if you're not logged in the button's text reads **Start** and takes you to Create a Linux virtual machine in Azure.

**Comment by JasonWHowell** [comment]
@tradexy 
Based on @cryophobia 's feedback, sounds like it could be that you or someone in the browser prior had made progress on the path, before the time when you clicked **Continue**, so it resumed at the module where you stopped. Othwerise, the page should have said **Start** instead of **Continue**. 

![image](https://user-images.githubusercontent.com/5067358/90173126-d34b2800-dd58-11ea-9b5c-b6477aa8c7e0.png)

![image](https://user-images.githubusercontent.com/5067358/90173144-db0acc80-dd58-11ea-873e-bd95ac7091ac.png)


If that's not the case, feel free to discuss further. 

Thanks, Jason
#please-close

**Comment by tradexy** [comment]
@cryophobia and @JasonWHowell - Agreed, good to close. Many thanks for looking this query and providing the explanation. By the way, the "Learn" content is great; technically, grammatically and captivating. Well done to the team!

**Comment by addisonkalven** [comment]
One of the best ways to nail a certification like Azure Administrator Associate is by preparing yourself through Microsoft Test Questions. The best Practice Questions can be availed from [Dumpspedia](https://www.dumpspedia.com) we have made many of your students successful throughout years of our service. With the most accurate and valid all around the world AZ-104 Practice Exam Questions, anybody can make it in Microsoft Azure Administrator Exam Exam. Joined by Professional Experts we have made sure our AZ-104 PDF Questions are flawless in every aspect. Help yourself to our website anytime in 24 hours of a day.
